docs/api/en/animation/AnimationAction.html

```diff
@@ -13,7 +13,7 @@ <h1>[name]</h1>
 			AnimationActions schedule the performance of the animations which are stored in
 			[page:AnimationClip AnimationClips].<br /><br />
 
-			Note: Most of AnimationAction's methods can be chained.<br /><br />
```

Comment:
Do you mind explaining why this wrong? The sentence refers to the methods of `AnimationAction`. So the `'s` means `its`.